keel/Core/err_log.py

```diff
@@ -0,0 +1,16 @@
+
+import logging
+import json
+
+logger = logging.getLogger('app-logger')
+
+def log_error(severity, api_loc, user_id, **kwargs):
+
+	logger.error( str(severity) + " - " + \
```

Comment:
already using logger.error
which defines the serverity of the error
so would the severity argument be useful here

Reply:
this format is already defined in logging engine in settings.py
kindly check LOGGING['formatters'] in settings.py